types/stripe/index.d.ts

```diff
@@ -299,35 +286,224 @@ declare namespace Stripe {
+                tax_id_registrar?: string;
+
+                /**
+                 * Whether the company’s business VAT number was provided.
+                 */
+                vat_id_provided?: boolean;
+            };
 
             /**
-             * Account-level settings to automatically decline certain types of charges
-             * regardless of the bank’s decision.
+             * Information about the person represented by the account.
+             * This field is null unless business_type is set to individual.
              */
-            decline_charge_on?: {
+            individual?: {
+                /**
+                 * The individual’s primary address.
+                 */
+                address?: IAddress;
+
+                /**
+                 * The Kana variation of the the individual’s primary address (Japan only).
+                 */
+                address_kana?: IAddressKana;
+
+                /**
+                 * The Kanji variation of the the individual’s primary address (Japan only).
+                 */
+                address_kanji?: IAddressKanji;
+
+                /**
+                 * The individual’s date of birth.
+                 */
+                dob?: {
+                    /**
+                     * The day of birth, between 1 and 31.
+                     */
+                    day: number;
+                    /**
+                     * The month of birth, between 1 and 12.
+                     */
+                    month: number;
+                    /**
+                     * The four-digit year of birth.
+                     */
+                    year: number;
+                }
+
+                /**
+                 * The individual's email address.
+                 */
+                email?: string;
+
+                /**
+                 * The individual’s first name.
+                 * This can be unset by updating the value to null and then saving.
+                 */
+                first_name?: string;
+
+                /**
+                 * The Kana variation of the the individual’s first name (Japan only).
+                 * This can be unset by updating the value to null and then saving.
+                 */
+                first_name_kana?: string;
+
+                /**
+                 * The Kanji variation of the individual’s first name (Japan only).
+                 * This can be unset by updating the value to null and then saving.
+                 */
+                first_name_kanji?: string;
+
+                /**
+                 * The individual’s gender (International regulations require either “male” or “female”).
+                 * This can be unset by updating the value to null and then saving.
+                 */
+                gender?: "male" | "female";
+
+                /**
+                 * Whether the individual’s personal ID number was provided.
+                 */
+                id_number_provided?: boolean;
```

Comment:
As mentioned in my previous comment, these `x_provided` properties can't be added to the `IAccountShared` interface as `IAccountUpdateOptions` and `IAccountCreationOptions` extend it and they aren't valid props on that create method.

`x_provided` props should be added to the `IAccount` interface and `x` props should be added to the `IAccountUpdateOptions` interface.